### PR TITLE
feat(i18n): RU localization

### PR DESCRIPTION
**Pull request by arnobase**
Russian localization
add public/locales/ru
edit 'languages.json' in [en, fr, cn]
edit i18n.ts

### CODE REVIEW

File: public/locales/ru/translation.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    "text": "Не волнуйтесь, мы сейчас создаем действительно хорошее расширение,",
+    "text2": "а пока мы рекомендуем скачать Polkadot.js"
+  },
+  "Install Polkadot.js Extension": "Установите расширение Polkadot.js",
+  "Requires Configuration": "Требуется конфигурация",
+  "desktopRequired": {
+    "title": "Извините!",
+    "subtitle": "Эта функция в настоящее время доступна только на настольных компьютерах, но мы скоро запустим версию для мобильных устройств.",
+    "text": "А пока не стесняйтесь проверить нашу панель управления Crowdloans",
+    "primaryCta": "Просмотр Crowdloans"
```
Comment (juliuscarvajal) [inline public/locales/ru/translation.json:40]: `translation.json` had a new field:
```
  "No wallet found": "No wallet found"
```

Would be good to add them in. Cheers.

Comment (juliuscarvajal) [reply public/locales/ru/translation.json:40]: Also this one:
```
  "Contributed": "Contributed",
```

File: public/locales/ru/banners.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  "buy": {
+    "header": "купить DOT/KSM",
+    "description": "Покажите мне деньги",
+    "primaryCta": "Купить"
+  },
+  "unlockTalisman": {
+    "header": "Разблокируйте талисман",
+    "description": "Внесите вклад в любой коллективный заем и получите Spirit Key, он понадобится вам позже ...",
+    "primaryCta": "Подробнее"
+  }
```
Comment (juliuscarvajal) [inline public/locales/ru/banners.json:26]: New field:
```
  "noWallet": {
    "header": "Oh no!",
    "description": "It doesnt look like you’ve got a wallet extension installed. We recommend downloading Polkadot.js",
    "primaryCta": "Download Polkadot.js"
  }
```

File: public/locales/ru/nav.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+  "Portfolio": "Портфолио",
+  "Crowdloans": "Crowdloans",
+  "Buy DOT/KSM": "купить DOT/KSM",
+  "Buy": "Купить",
+  "Request Features": "Запросить функции",
+  "GitHub": "GitHub",
+  "Discord": "Discord",
+  "Twitter": "Twitter",
+  "Medium": "Medium"
```
Comment (juliuscarvajal) [inline public/locales/ru/nav.json:10]: New fields:
```
  "Terms of use": "Terms of use",
  "Privacy policy": "Privacy policy"
```

**Comment by juliuscarvajal** [comment]
Thanks @arnobase just few new missing translations. Cheers.

File: public/locales/ru/translation.json (hunk 1 above)
Comment (arnobase) [reply public/locales/ru/translation.json:41]: done

File: public/locales/ru/banners.json (hunk 2 above)
Comment (arnobase) [reply public/locales/ru/banners.json]: done

File: public/locales/ru/nav.json (hunk 3 above)
Comment (arnobase) [reply public/locales/ru/nav.json]: ok too

**Comment by arnobase** [comment]
added new commit #58aa5a2d1987baa62dc14108848b93b384a139ff

**Review by juliuscarvajal — APPROVED** [review]
Beautiful. Thanks.

**Pull request merged**